Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-GDPR.controller.ts

```diff
@@ -173,8 +201,14 @@
                     }
                     else {
                         itSystemUsageService.removeDataLevel(itSystemUsage.id, dataLevel)
-                            .then(onSuccess => notify.addSuccessMessage("Feltet er opdateret!"),
-                                onError => notify.addErrorMessage("Kunne ikke opdatere feltet!"));
+                            .then(onSuccess => {
+                                notify.addSuccessMessage("Feltet er opdateret!");
+                                if (dataLevel === Kitos.Models.ViewModel.ItSystemUsage.SensitiveDataLevelViewModel.levels.personal.value) {
+                                    $scope.usage.personalData = [];
+                                    $scope.personalData.options.forEach(option => option.checked = false);
```

Comment:
why assume this responsibility here?. I would think the right and more robust approach would be to get the new dto from the endpoint and update the data source of the checkboxes ... this implementation assumes specific business logic